src/sounds_generation.py

```diff
@@ -0,0 +1,39 @@
+import math
+
+import parselmouth
+
+import music
+
+
+def frequency_to_note(frequency):
+    '''
+    A4 440 Hz
+    '''
+    if frequency == 0:
+        return None
+    return round(12*math.log2(frequency/440.0)+45) % 12
+
+
+def get_sounds_from_file(file):
+    sounds = []
+    snd = parselmouth.Sound(str(file))
+    pitch = snd.to_pitch()
+    frequencies = pitch.selected_array['frequency']
+    notes = [frequency_to_note(freq) for freq in frequencies]
+    return get_sounds_from_list(pitch.xs(), notes)
+    
+
+def get_sounds_from_list(pitch, notes):
```

Comment:
should be
get_sounds_from_list(timestamps, notes)
Pitch is the height of the tone and doesn't make sense here.